frontend/packages/ux-editor/src/components/config/editModal/EditOptions/OptionTabs/EditTab/OptionListEditor/hooks/useConcatOptionsLabels.test.ts

```diff
@@ -0,0 +1,56 @@
+import type { TextResource } from '@studio/components-legacy';
+import type { Option } from 'app-shared/types/Option';
+import { useConcatOptionsLabels } from './useConcatOptionsLabels';
+import { textMock } from '@studio/testing/mocks/i18nMock';
+
+// Test data:
+const textResources: TextResource[] = [
+  {
+    id: 'text-with-value',
+    value: 'some-text',
+  },
+  {
+    id: 'empty-text',
+    value: '',
+  },
+];
+
+describe('useConcatOptionsLabels', () => {
+  it('returns a string of labels with no textResources provided', () => {
+    const optionList: Option[] = [
+      {
+        value: 'value',
+        label: 'random-text',
+      },
+    ];
+    const result = useConcatOptionsLabels(optionList);
+    expect(result).toEqual(optionList[0].label);
+  });
+
+  it('return a string of labels translated from textResources', () => {
+    const optionList: Option[] = [
+      {
+        value: 'value',
+        label: 'text-with-value',
+      },
+    ];
```

Comment:
Jeg tenker at en liste i testdataen burde inneholde minst to instanser, i hvert fall i dette tilfellet, siden det er en "happy path". Da får vi testet med en faktisk liste, og vi får inkludert ting som at `|`-tegnet kommer med. Hva tenker du?